lib/rust/parser/src/syntax/operator.rs

```diff
@@ -133,31 +136,30 @@ fn resolve_operator_precedence_internal(items: Vec<syntax::Item<'_>>) -> syntax:
                     // Prev operator in the [`operator_stack`] has a higher precedence.
                     let lhs = output.pop().map(|t| t.to_ast());
                     if lhs.is_none() { was_section_used = true; }
-                    let ast = syntax::Tree::opr_app(lhs, prev_opr.elem, Some(rhs.to_ast()));
+                    let ast = syntax::tree::apply_operator(lhs, prev_opr.elem, Some(rhs.to_ast()));
```

Comment:
Most of the items in tree are named as nouns, like "TypeDefinition". In fact, I've never seen AST that would use non-noun names. This should be named `operator_application` instead. (pls apply to other changed names).

Reply:
This is a helper function. It implements the rule that, in the RHS of an `OprApp`, a non-operator block is interpreted as a `BodyBlock` (whereas in other positions, it would be an argument block). Likewise, `apply` creates an `App`, `ArgumentBlockApplication`, or `OperatorBlockApplication`.